deploying.md

```diff
@@ -0,0 +1,113 @@
+# Deploying Philanthropy Data Commons software
+
+This document guides one through deployment for the back-end services at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has its own process.
+
+What gets deployed? A [docker compose yaml file][1].
+
+How does it get deployed? A [bash script running under cron][2] pulls the new,
+stops the old, and starts the new compose file.
+
+How does the deploy script know how what to deploy? It [looks inside a file][3].
+
+This setup supports both automated and manual deployments that have exactly the
+same steps with exactly the same code.
+
+[1]: https://github.com/PhilanthropyDataCommons/deploy/blob/main/compose.yml
+[2]: https://github.com/PhilanthropyDataCommons/deploy/blob/main/deploy.sh
+[3]: https://github.com/PhilanthropyDataCommons/deploy/blob/e0d24b8a19ee563938f8211bca0485a97ff1a6e9/deploy.sh#L90
+
+## The common case: deploy a newly built version of the PDC service
+
+This is the streamlined case. It assumes that both the test and production
+machines were already set up with users, directories, docker, certbot, scripts,
+environment variables, additional keycloak jar files, and so forth.
+
+### Optionally review this background information
+
+On a merge to the main branch in the [service repository][4], tests run and a
```

Comment:
"When a branch is merged into `main` in the service repository, GitHub Actions will automatically run the test suite and then, assuming the tests succeed, build a new docker image containing the code and all its dependencies.  That image will be tagged with a tag like `give-an-actual-example-tag-here`. The tag is on the binary image, not on the source code, but the tag's text is based on the git commit, so that we can always trace the image back to the source code whence it came."

That wording is just a suggestion.  The main thing is to steer clear of subjectless voice as much as possible in this kind of documentation.  Always say exactly what actor performs what actions, with what results, in what order.

In the rest of the document, you seem to mostly use the latter style, and it's good IMHO.